views/default/user.html

```diff
@@ -19,6 +24,10 @@ <h2>
         {{pass}}
     </div>
     <div id="web2py_user_form" class="col s8">
+    <a href="/default/googleauth">
+        <img src="{{=URL('static','images/btn_google_sigin.png')}}" height="50px">
```

Comment:
Can we use the Google documentation's html instead of the image?
They have a bunch of guidelines to adhere to which is better if they update on their own with minimal effort from our side.

Reply:
https://developers.google.com/identity/sign-in/web/build-button

Reply:
the button not at all looks good and the button used is downloaded from google documentation only

Reply:
Even if we want to go ahead with hardcoded image we will need separate buttons for sign in and sign up